app/controllers/application.js

```diff
@@ -30,8 +30,8 @@ export default Ember.Controller.extend({
     return /Android/i.test(navigator.userAgent) && !this.get("isMobileApp");
   }),
 
-  isAndroidBrowser: Ember.computed("isMobileApp", function() {
-    return /Android/i.test(navigator.userAgent) && !this.get("isMobileApp");
+  isBrowsePage: Ember.computed("currentPath", function() {
+    return ["browse", "package_category"].indexOf(this.get("currentPath")) >= 0;
```

Comment:
how come `isBrowsePage` is true if the path is `package_category` ? 🤔

Reply:
browse_related i meant.. by this, will change the name.